cypress/integration/Add_Item_Button.js

```diff
@@ -0,0 +1,7 @@
+describe("Properly calculates height of AddItemButton component", () => {
+    it('properly reads the height prop of AddItemButton component and sets it accordingly ', () => {
+        cy.visit('/AddItemButtonTest');
+        cy.get('.makeStyles-addNewItemStyles-3').should('have.css', 'height', '240px');
```

Comment:
don't use css classes to select for components. use the data-cy attribute. 

Reference: https://docs.cypress.io/guides/references/best-practices.html#Selecting-Elements